harness/determined/cli/checkpoint.py

```diff
@@ -145,6 +145,12 @@ def delete_checkpoints(args: Namespace) -> None:
                     action="store_true",
                     help="Only print the path to the checkpoint.",
                 ),
+                Arg(
+                    "-m",
```

Comment:
`-m` is already a global flag for setting the master address; let's not add another `-m` with another meaning in a different scope

Reply:
Sure, I don't think we need its short form.